Gemfile

```diff
@@ -1,8 +1,14 @@
 source 'https://rubygems.org'
 
-gem "github-pages", '193', group: :jekyll_plugins
+gem 'html-proofer'
```

Comment:
You resolved the conversation, but I am still +1 to remove this. It's going to make local development harder for existing and new users, and we don't need to run html-proofer.

Reply:
@vsoch  I actually forgot to discuss this point. `The typical user with ruby / jekyll will need to bundle install again`

After this PR is merged, the user needs to run `bundle install` to run this project locally since the Jekyll version has been updated. 

> we don't need to run html-proofer

I am happy with removing `html-proofer` if you think soo.

Reply:
Yes, that is what I was suggesting.

Reply:
@vsoch Removed `html-proofer` 👍